perun-core/src/test/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntryIntegrationTest.java

```diff
@@ -2075,6 +2079,64 @@ public void getEnrichedResourcesForFacilityWithGivenAttributes() throws Exceptio
 		assertThat(eResources.get(0).getAttributes().get(0).getName()).isEqualTo(A_R_C_ID);
 	}
 
+	@Test
+	public void getResourceAssignments() throws Exception {
+		System.out.println(CLASS_NAME + "getResourceAssignments");
+
+		vo = setUpVo();
+		member = setUpMember(vo);
+		group = setUpGroup(vo, member);
+		facility = setUpFacility();
+		resource = setUpResource();
+
+		resourcesManager.assignGroupToResource(sess, group, resource);
+
+		List<AssignedResource> resources = resourcesManager.getResourceAssignments(sess, group, null);
+		AssignedResource expectedResource = new AssignedResource(new EnrichedResource(resource, null), GroupResourceStatus.ACTIVE);
+
+		assertThat(resources.size()).isEqualTo(1);
+		assertThat(resources).containsExactly(expectedResource);
+		assertThat(resources.get(0).getEnrichedResource().getAttributes())
+			.containsExactlyInAnyOrderElementsOf(perun.getAttributesManager().getAttributes(sess, resource));
+	}
+
+	@Test (expected=GroupNotExistsException.class)
+	public void getResourceAssignmentsWhenGroupNotExists() throws Exception {
```

Comment:
Instead of this, you can use the `assertThatExceptionOfType().isThrownBy()` from the assertj library. It makes it more clear where the exception is thrown and it provides several useful methods.

Reply:
Replaced